biojava-genome/src/main/java/org/biojava/nbio/genome/io/fastq/AbstractFastqReader.java

```diff
@@ -18,7 +18,7 @@
  *      http://www.biojava.org/
  *
  */
-package org.biojava.nbio.sequencing.io.fastq;
+package org.biojava.nbio.genome.io.fastq;
```

Comment:
Should all these classes moved to a new package have `@since 5.0.0` tags?  Note there are other `@since` tags present on classes and methods throughout the old `o.b.n.sequencing.io.fastq` package.

Reply:
Good point, I'm not sure how to do `@since` tags for these cases where something existed but was moved, rather than appeared new. Ideas anyone?

Reply:
I would not use the `@since` tag, because it means that the class is new. Maybe just add a line like:

`moved from org.biojava.nbio.sequencing in 5.0.0`